molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -88,7 +88,8 @@
         <script src="<@resource_href "/js/component/SelectBox.js"/>"></script>
         <script src="<@resource_href "/js/component/Spinner.js"/>"></script>
         <script src="<@resource_href "/js/component/Table.js"/>"></script>
-        <script src="<@resource_href "/js/component/TextArea.js"/>"></script>     
+        <script src="<@resource_href "/js/component/TextArea.js"/>"></script>
+        <script src="<@resource_href "/js/component/IdcardBiobankIndexerComponent.js"/>"></script>
```

Comment:
Can we add this only for the idcard module? molgenis-core-ui should not be aware of this component.